server/apps/main/helpers.py

```diff
@@ -132,6 +133,46 @@ def get_oh_file(ohmember, uuid):
 
     return file[0]
 
+def get_oh_file(url):
```

Comment:
Would you be interesting in writing some tests for the new functions as well? I'm asking as I know that it can be a bit tricky as we need to have the "right" VCR cassettes to mock the external API responses! Also happy to help and walk you through it if that would be useful!

Reply:
I've given it a go. I'm not sure if the VCR tests I've added are along the right lines, so would value your input.

Reply:
Yep, that looks all right, well done!